tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -599,20 +599,20 @@ public static ExplorationTestDescription GetExplorationTestDescription(Explorati
             {
                 Name = ExplorationTestName.eShopOnWeb,
                 GitRepositoryUrl = "https://github.com/dotnet-architecture/eShopOnWeb.git",
-                GitRepositoryTag = "netcore2.1",
+                GitRepositoryTag = "main",
```

Comment:
I know we want to target a specific tag/version to make sure there is no variability in the CI from distant changes, but here the repo has been archived since 6+ months, so I think we're pretty safe pulling main, it shouldn't change any time soon.

Reply:
Even so, it probably makes sense to specify the specific commit? 
```suggestion
                GitRepositoryTag = "2414014bfa0f4d2021b5bc9061429a98d232f440",
```

Reply:
thanks, I didn't know we could specify a commit hash here

Reply:
actually we can't, this breaks the command used for cloning. Also, it's not even a feature that's readily available with git, it depends on server side config (see https://stackoverflow.com/a/3489576/787883).
I implemented it using a second command to `git reset --hard` to a provided sha.